harness-delegate-ng/templates/secret.yaml

```diff
@@ -8,4 +8,4 @@ metadata:
 type: Opaque
 data:
   # Base 64 encoded account secret
-  DELEGATE_TOKEN: {{ .Values.delegateToken | b64enc | quote }}
```

Comment:
please update chart.yaml, index.yaml and update packaged chart in charts directory.